### DISCUSSION TITLE
Advanced filters

### DISCUSSION THREAD

**Discussion by adrianthedev** [avo-hq/avo#739] [Feature requests]
We need an automated more advanced filtering system.

![image](https://user-images.githubusercontent.com/1334409/159448133-60edb2c8-fdca-4250-9571-5ce650aecad2.png)

**Comment by brandonzylstra** [comment]
"does not contain" would be much more natural than "not contains"

**Reply by adrianthedev** [reply]
That's what we ended up with. You can test it [here](https://main.avodemo.com/avo/resources/users).
<img width="381" alt="CleanShot 2023-07-11 at 10 45 01@2x" src="https://github.com/avo-hq/avo/assets/1334409/7525484a-932e-48a9-b19e-dcb0b7da1b17">

Thanks for the feedback!

**Comment by Paul-Bob** [comment]
https://docs.avohq.io/3.0/dynamic-filters.html
